docs/concepts/aws.md

```diff
@@ -0,0 +1,26 @@
+# AWS
+
+AWS is one of the cloud platforms supported by Lokomotive. This document explains various architecture decisions specific to this platform.
```

Comment:
```suggestion
AWS is one of the cloud platforms supported by Lokomotive. This document explains various architecture decisions and details specific to this platform.
```

These "platform" docs will likely also explain **how things work** and not only what decisions we've taken and why we've taken them.